### PR TITLE
chore: Remove import react

### PR DESCRIPTION
**Pull request by memoyil**

### CODE REVIEW

File: src/components/EasterEgg/EasterEgg.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { useState, useCallback } from 'react'
+import { useState, useCallback, memo } from 'react'
 import { FallingBunnies, FallingBunniesProps, useKonamiCheatCode } from '@pancakeswap/uikit'
 
 const EasterEgg: React.FC<FallingBunniesProps> = (props) => {
```
Comment (chef-jojo) [inline src/components/EasterEgg/EasterEgg.tsx:4]: I didn't know we can do this 🤯 maybe types import globally

Comment (memoyil) [reply src/components/EasterEgg/EasterEgg.tsx:4]: React namespace is imported thats why we can use without import

**Review by chef-jojo — CHANGES_REQUESTED** [review]
fix build

**Comment by memoyil** [comment]
> fix build

Done

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**